applications/openshift/kubelet/kubelet_configure_event_creation/rule.yml

```diff
@@ -9,11 +9,17 @@ description: |-
     reconstucting the chain-of-events.
 
     To prevent log creation limiting, edit the kubelet configuration
+{{%- if product == "ocp4" %}}
+    file <tt>/etc/kubernetes/kubernetes.conf</tt>
```

Comment:
Editing a config file directly on a node is not permanent, these tunables would have to be exposed through the cluster's API (i.e., for general node configuration, that is through MachineConfigs) in order for them to stick and also be in effect for all nodes.

Reply:
@mrogers950 do you know a way to edit this configuration from the API side?

Reply:
While this will be done through machine config, many users will understand that it is not configurable on the API side. What's probably going to make sense is to get all the rules into a profile, and then figure out how to correctly split out the rules that are node only vs API.

Reply:
> While this will be done through machine config, many users will understand that it is not configurable on the API side. What's probably going to make sense is to get all the rules into a profile, and then figure out how to correctly split out the rules that are node only vs API.

Sounds good to me.